src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,8 +1,112 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import './ProductDetail.scss';
 
 const ProductDetail = () => {
-  return <div>Product Detail</div>;
+  const [productDetailData, setProductDetailData] = useState({});
+  const params = useParams();
+
+  useEffect(() => {
+    fetch('/data/teaListData.json')
+      .then(response => response.json())
+      .then(data => setProductDetailData(data));
+  }, [params.id]);
+
+  useEffect(() => {
+    fetch(`http://10.58.52.228:8002/items/${params.id}`)
+      .then(response => response.json())
+      .then(data => setProductDetailData(data));
```

Comment:
```suggestion
      .then(data => setProductDetailData(data.items[0][params.id - 1]));
```

요청시에 아래에서 작성하는대로 그대로 넣어주면 아래에서는 키 값만 사용할 수 있겠습니다!